### ISSUE TITLE
Integration with field service SaaS

### ISSUE THREAD

**Issue by rajesh99** [YourAverageTechBro/social-queue#85]
I have a field service SaaS.  Can this be integrated at the source code level with my field service saas 
Benefits for the companies
- Allow to schedule posts within the saas
- Allow their customer to share the positive customer experience provided by the technician

**Comment by YourAverageTechBro** [comment]
Yeah since it is open source you could do whatever you want with it if you integrate it into your SaaS 🙂

Alternatively if you would like me to open up API access on my hosted version, you can email me at dohyun@youraveragetechbro.com.
